region/tourian/main/Metroid Room 1.json

```diff
@@ -281,7 +281,9 @@
       },
       "requires": [
         "f_KilledMetroidRoom1",
-        "h_canArtificialMorphCeilingBombJump"
+        "h_canArtificialMorphCeilingBombJump",
```

Comment:
This strat was already here so perhaps a comment for later, but I don't think you can get past the center rinka.  
With the Morph item strat you jump to the floating platforms first.
A diagonal bombjump looks like you could make it with some luck.

[2,1] could HBJ to the floating platform and ceiling/diagonal IBJ from there. And HBJ from the left platform to end.  A Purely ceiling IBJ wouldn't get past the double Rinkas.

Reply:
The Rinkas shouldn't spawn once killed once with Bombs

Reply:
You are right, pseudo GMode treated them incorrectly.